cirq-rigetti/cirq_rigetti/service_test.py

```diff
@@ -11,23 +9,3 @@ def test_get_rigetti_qcs_service():
     through `pyquil.get_qc`."""
     service = get_rigetti_qcs_service('9q-square', as_qvm=True, noisy=False)
     assert service._quantum_computer.name == '9q-square-qvm'
-
-
-@pytest.mark.rigetti_integration
-def test_rigetti_qcs_service_api_call():
```

Comment:
The client is no longer httpx-based and is now [implemented in rust](https://github.com/rigetti/qcs-sdk-rust/blob/main/crates/python/src/client.rs).